setup.py

```diff
@@ -36,6 +37,7 @@
     'js.jquery_timepicker_addon',
     'js.jqueryui>=1.8.24',
     'js.jqueryui_tagit',
+    'kotti_tinymce',
```

Comment:
``kotti_tinymce`` is not a requirement for ``Kotti``. It is however needed to run the included ``*.ini``s. That's why it's in ``requirements.txt`` but not in ``setup.py``.